pg_search/src/api/index.rs

```diff
@@ -256,81 +253,79 @@ pub fn fuzzy_phrase(
     prefix: default!(Option<bool>, "NULL"),
     match_all_terms: default!(Option<bool>, "NULL"),
 ) -> SearchQueryInput {
-    let (field, path) = split_field_and_path(&field);
     SearchQueryInput::FuzzyPhrase {
-        field,
+        field: field.into_inner(),
         value: value.expect("`value` argument is required"),
         distance: distance.map(|n| n as u8),
         transposition_cost_one,
         prefix,
         match_all_terms,
-        path,
     }
 }
 
 #[pg_extern(name = "more_like_this", immutable, parallel_safe)]
 pub fn more_like_this_empty() -> SearchQueryInput {
-    panic!("more_like_this must be called with either with_document_id or with_document_fields");
+    panic!("more_like_this must be called with either document_id or document_fields");
 }
 
 #[allow(clippy::too_many_arguments)]
 #[pg_extern(name = "more_like_this", immutable, parallel_safe)]
 pub fn more_like_this_fields(
-    with_document_fields: String,
-    with_min_doc_frequency: default!(Option<i32>, "NULL"),
-    with_max_doc_frequency: default!(Option<i32>, "NULL"),
-    with_min_term_frequency: default!(Option<i32>, "NULL"),
-    with_max_query_terms: default!(Option<i32>, "NULL"),
-    with_min_word_length: default!(Option<i32>, "NULL"),
-    with_max_word_length: default!(Option<i32>, "NULL"),
-    with_boost_factor: default!(Option<f32>, "NULL"),
-    with_stop_words: default!(Option<Vec<String>>, "NULL"),
+    document_fields: String,
+    min_doc_frequency: default!(Option<i32>, "NULL"),
+    max_doc_frequency: default!(Option<i32>, "NULL"),
+    min_term_frequency: default!(Option<i32>, "NULL"),
+    max_query_terms: default!(Option<i32>, "NULL"),
+    min_word_length: default!(Option<i32>, "NULL"),
+    max_word_length: default!(Option<i32>, "NULL"),
+    boost_factor: default!(Option<f32>, "NULL"),
+    stop_words: default!(Option<Vec<String>>, "NULL"),
 ) -> SearchQueryInput {
```

Comment:
Renaming these arguments is a user-facing breaking API change. 

I don't believe it should be included in this PR.

Reply:
Undoing this will then change the upgrade.sql script too.

Reply:
@rebasedming and I discussed this last week, it was an intentional change, but I don't feel strongly about it. At @rebasedming would you rather keep the `with_` names instead?

Reply:
We can call them whatever you'd like in the json since that's new, but changing the function argument names has the potential to break existing user queries plus these ORM tools that are starting to provide pg_search wrappers.

If we insist on changing them, we'll need to move to v0.12.0 because of this PR.

Reply:
I think it's still okay to do breaking changes. If we think this is important, let's do it and move to `0.12.0`.

Reply:
Let's do `0.12.0` next 👍